src/widgets/adhole/AdHoleControls.tsx

```diff
@@ -0,0 +1,106 @@
+import { Badge, Button, Card, Group, Image, Stack, Text } from '@mantine/core';
+import { IconDeviceGamepad, IconPlayerPlay, IconPlayerStop } from '@tabler/icons';
+import { useConfigContext } from '../../config/provider';
+import { defineWidget } from '../helper';
+import { WidgetLoading } from '../loading';
+import { IWidget } from '../widgets';
+import { useAdHoleControlMutation, useAdHoleSummeryQuery } from './query';
+import { PiholeApiSummaryType } from './type';
+
+const definition = defineWidget({
+  id: 'adhole-controls',
+  icon: IconDeviceGamepad,
+  options: {},
+  gridstack: {
+    minWidth: 2,
```

Comment:
I noticed that on medium displays, having the width at 2 would cause text and icons to be cut-off. Maybe having the minWidth at 3 might help
```suggestion
    minWidth: 3,
```